src/ProcessManager.php

```diff
@@ -141,11 +89,11 @@ class ProcessManager
     protected $inRestart = false;
 
     /**
-     * The number of seconds to wait before force closing a worker during a reload.
+     * The debug timer
```

Comment:
Can we make a little more clear what that debug timer does, e.g. code hot reload timer?